### ISSUE TITLE
[Feature Request] Consider putting a link to the Wiki in the README

### ISSUE THREAD

**Issue by owenconti** [lhorie/mithril.js#1360]
Not a lot of repositories utilize the Wiki feature as well as Mithril does. Because of that, I believe it is not common for a developer new to a framework to check the Wiki inside the repo. 

I started to learn Mithril a couple of days ago, mostly via the documentation on the main website: http://mithril.js.org/

I wanted to look at some example apps, tutorials, etc based around Mithril so I did some searching but came up relatively empty considering the project has 6k+ stars. I tried Google, Twitter, Youtube, etc.

It took me until today (3 dev sessions) to find the Wiki. I found it via a random Github issue while I was debugging a problem. 

TLDR: 

Placing a link to the Wiki from the README (main page of the repo) would be _extremely_ beneficial to newcomers. I would be more than happy to submit a PR for it.

**Comment by lhorie** [comment]
PR would most definitely be welcome
